### ISSUE TITLE
Accessing factories like React.DOM.input has been deprecated

### ISSUE THREAD

**Issue by afhole** [YouCanBookMe/react-datetime#360]
React/ReactDOM 15.6.1 is giving me a deprecation warning:
```Accessing factories like React.DOM.input has been deprecated and will be removed in v16.0+. Use the react-dom-factories package instead.  Version 1.0 provides a drop-in replacement. For more info, see https://fb.me/react-dom-factories```

```
printWarning	@	lowPriorityWarning.js:40
lowPriorityWarning	@	lowPriorityWarning.js:59
React.DOM.(anonymous function)	@	React.js:123
render	@	DateTime.js:412
```

**Comment by studyingMole** [comment]
Looks like this might be the culprit?

[https://github.com/YouCanBookMe/react-datetime/blob/master/DateTime.js#L412](https://github.com/YouCanBookMe/react-datetime/blob/master/DateTime.js#L412)

**Comment by dominikdosoudil** [comment]
@gwing33 has made a PR that should fix it #353 :)

**Comment by layneanderson** [comment]
Closing as fix has been merged.
